sphinxcontrib/images.py

```diff
@@ -265,13 +265,15 @@ def download_images(app, env):
                       .format(src, dst))
 
 
-def configure_backend(app):
+def configure_backend(app, main_config):
     global DEFAULT_CONFIG
 
     config = copy.deepcopy(DEFAULT_CONFIG)
     config.update(app.config.images_config)
     app.config.images_config = config
 
+def builder_init(app):
```

Comment:
This is still configuring the backend and probably shouldn't be renamed

```suggestion
def configure_backend(app):
```